calyx-py/calyx/builder_util.py

```diff
@@ -145,13 +159,27 @@ def insert_decr(comp: cb.ComponentBuilder, reg, cellname, val=1):
     return decr_group
 
 
-def mem_load(comp: cb.ComponentBuilder, mem, i, reg, group):
-    """Loads a value from one memory into a register.
+def insert_reg_store(comp: cb.ComponentBuilder, reg, val, group):
```

Comment:
I'm starting to think we can quickly build a DSL in python with methods like this. Specifically, this corresponds to `r := v` so we can define a DSL where `:=` is overloaded to generate this method instead. 

Nothing to address here. Just seeding some thoughts